flycheck.el

```diff
@@ -1556,7 +1572,16 @@ A checker is registered if it is contained in
 
 A checker is disabled if it is contained in
 `flycheck-disabled-checkers'."
-  (memq checker flycheck-disabled-checkers))
+  (or (memq checker flycheck-disabled-checkers)
+      (memq checker flycheck-automatically-disabled-checkers)))
+
+(defun flycheck-automatically-disabled-checker-p (checker)
+  "Determine whether CHECKER has been automatically disabled.
+
+A checker has been automatically disabled if it is contained in
+`flycheck-automatically-disabled-checkers'."
+  (memq checker flycheck-automatically-disabled-checkers))
```

Comment:
Do you want to use that function in flycheck-disabled-checker-p?

Reply:
Good catch.